packages/genesys-spark-components/src/components/stable/gux-content-search/gux-content-search.scss

```diff
@@ -16,6 +16,7 @@
   flex-basis: 100%;
   flex-shrink: 1;
   width: 100%;
+  min-width: 20px;
```

Comment:
This should have a token, maybe add to GDS-2451

Reply:
Updated that ticket to include this ask.